### PR TITLE
zDNN conda package

### PR DESCRIPTION
**Pull request by pradghos**
## Checklist before submitting

- [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

conda package for IBM Z Deep Neural Network Library

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```
Comment (npanpaliya) [inline recipe/build.sh]: Please add copyright info.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: zdnn 
+  version: {{ version }} 
+
+source:
+  git_url: https://github.com/IBM/zDNN.git
+  git_tag: v{{ version }}
+
+build:
+  number: 0
```
Comment (npanpaliya) [inline recipe/meta.yaml]: We start build numbers from 1.

**Comment by npanpaliya** [comment]
Also, please update README similar to other feedstocks. Need to add github actions/workflows too.

**Comment by pradghos** [comment]
Thanks @npanpaliya , I have addressed review comments in latest commit.

File: recipe/build.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *****************************************************************
+
+autoconf
+
+CFLAGS_INIT="-march=z9-109" LD=$CC ./configure --prefix="${PREFIX}"
```
Comment (ravigumm) [inline recipe/build.sh:20]: Are we using march=z9-109 instead of march=z14 because of trying to support older z versions ?

Comment (pradghos) [reply recipe/build.sh:20]: Yes, `CFLAGS_INIT` is only used to build `zdnn_init.c`. main functionality of the library is built with `CFLAGS` which is using `-march=z14`.

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**